pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -713,6 +713,7 @@ func (e *cloudWatchExecutor) getAllMetrics(region string) (cloudwatch.ListMetric
 
 	plog.Debug("Listing metrics pages")
 	var resp cloudwatch.ListMetricsOutput
+	plog.Debug("Listing metrics pages")
```

Comment:
This debug message is the same as 2 lines above.

Reply:
@papagian Ooops, thanks, it's left over from a merge.